Feat: 방 관리 아이템리스트 표시/수정하기 페이지 이동

JJongsKim opened the pull request:
## ⛳️ 작업 내용

/products/user-products API가 나와 연결하였습니다.

아예 Navigation bar 컴포넌트에서 텍스트 클릭 시 동작할 수 있도록 하는 게 좋을 것 같아
해당 컴포넌트에서 API 호출 후, atom값으로 넘겨 메인페이지 컴포넌트로 불러왔습니다.
그래서 방 관리를 클릭하지 않을 시, 메인페이지에서 기본으로 불러온 아이템 목록이 보이고 방 관리 텍스트를 클릭 시 유저가 올린 아이템 목록을 볼 수 있게 됩니다.
추가로 /Refrigation 페이지에서 수정하는 것도 함께 처리한다고 하셔서 톱니바퀴 클릭 시 해당 페이지로 이동하도록 경로를 수정했습니다.

## 🌱 PR 포인트

test1아이디에 기본 경매 리스트랑 유저가 올린 경매방 관리 리스트에 있는 아이템들이 똑같아서 헷갈리는데, 잘 불러와지는 것 같습니다!
props는 어떻게 넘기실지 로직을 짜놓은 게 있으실 것 같아서 우선 이동하는 것까지만 처리해두었습니다.

## 📸 스크린샷
|API연결 후 페이지 플로우|
|:--:|
|![수정페이지이동](https://user-images.githubusercontent.com/81777778/232227192-09f58ecd-f5c2-4006-baab-c867e62286c1.gif)|

## 📎 레퍼런스

<!-- 참고한 레퍼런스가 있다면 기록해주세요 -->

harksu committed:
feat: read products

harksu committed:
feat: create image

JJongsKim committed:
:sparkles: feat: 목업데이터로 방관리 구현

- 목업데이터로 이동 가능하게 수정
- 수정페이지로 이동하는 버튼도 방 관리 눌렀을 때만 보이도록 수정

JJongsKim committed:
:lipstick: style: 삭제버튼 UI 추가

JJongsKim committed:
:bug: fix: offset 이슈 해결

- 방 관리 누를 시 0으로 다시 초기화시켜 수정완료

harksu commented on src/pages/EditPage/index.tsx:
```diff
@@ -0,0 +1,13 @@
+  <div style={{ color: 'white' }}>
+    <h1>경매물품 수정페이지</h1>
+    <h4>여기 Registration이랑 구조가 버튼 다른 거 빼고 똑같은데,</h4>
+    <h4>혹시 수정페이지도 그거랑 같은 페이지에서 작업되나요?</h4>
```
이걸 지금 어떻게 해야되나 고민중입니다..! 아마 처음에 렌더링 할 때, props여부로 나눌 것 같아요!

harksu reviewed (approved):
고생하셨습니다!

JJongsKim committed:
Merge branch 'feature/products-api' into feat/route-editpage

JJongsKim committed:
:bug: fix: instanceAPI 수정, .env 형태 통일을 위해 / 추가

JJongsKim committed:
:sparkles: feat: 방관리에서 수정페이지로 이동

- Refrigation 페이지로 이동하도록 수정(추후 props넘기기 필요)
- 방 관리 API 토큰필요로 인해 instanceAPI를 사용하여 get 호출
- Photo border-radius 추가

JJongsKim committed:
:fire: remove: 필요없는 페이지 제거

harksu commented on src/components/Main/ProductContainer/ItemBox.tsx:
```diff
@@ -1,45 +1,55 @@
+const ItemBox = ({ items, isClicked }: ItemBoxProps) => {
+  const navigate = useNavigate();
+
+  // 여기 Registration으로 이동시에 props도 넘겨줄 수 있도록 ?
```
이걸 제가 그대로 바톤 받으면 되겠네요 👍

harksu reviewed (approved):
좋습니당 :) 
instance 관련해서 정상적으로 작동하신다고 했으니까, merge 해주시면 바톤 받을게요!
